core/rio/n3/src/test/java/org/eclipse/rdf4j/rio/n3/N3ParserTest.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+package org.eclipse.rdf4j.rio.n3;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.io.StringReader;
+
+import org.eclipse.rdf4j.model.Statement;
+import org.eclipse.rdf4j.model.ValueFactory;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+import org.eclipse.rdf4j.rio.helpers.ParseErrorCollector;
+import org.eclipse.rdf4j.rio.helpers.SimpleParseLocationListener;
+import org.eclipse.rdf4j.rio.helpers.StatementCollector;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * @author tanisha
+ */
+public class N3ParserTest {
+
+	private N3Parser parser;
+
+	private ValueFactory vf = SimpleValueFactory.getInstance();
+
+	private final ParseErrorCollector errorCollector = new ParseErrorCollector();
+
+	private final StatementCollector statementCollector = new StatementCollector();
+
+	private final String prefixes = "@prefix ex: <http://example.org/ex/> . \n@prefix : <http://example.org/> . \n";
+
+	private final String baseURI = "http://example.org/";
+
+	private SimpleParseLocationListener locationListener = new SimpleParseLocationListener();
+
+	/**
+	 * @throws java.lang.Exception
+	 */
+	@Before
+	public void setUp() {
+		parser = new N3Parser();
+		parser.setParseErrorListener(errorCollector);
+		parser.setRDFHandler(statementCollector);
+		parser.setParseLocationListener(locationListener);
+	}
+
+	@Test
+	public void testParseEquals() throws IOException {
+		String data = prefixes + " ex:foo = ex:bar. ";
+
+		parser.parse(new StringReader(data), baseURI);
+
+		assertTrue(errorCollector.getWarnings().isEmpty());
+		assertTrue(errorCollector.getErrors().isEmpty());
+		assertTrue(errorCollector.getFatalErrors().isEmpty());
+
+		assertFalse(statementCollector.getStatements().isEmpty());
+		assertEquals(1, statementCollector.getStatements().size());
+
+		for (Statement st : statementCollector.getStatements()) {
+			System.out.println(st);
```

Comment:
Ah, only just now noticed this - instead of printing this to stdout can you use do another assertion here, to check that the statement is what we expect: `(ex:foo, owl:sameAs, ex:bar)`?